flashcards/backend/decks/tests/test_models_difficulty_level.py

```diff
@@ -0,0 +1,11 @@
+from ..models import DifficultyLevel
+
+
+def test_difficulty_level_gui_representation():
+    difficulty_level = DifficultyLevel(name="hard", value=1)
+    assert str(difficulty_level) == "hard"
+
+
+def test_difficulty_level_custom_save_method(difficulty_level):
+    assert DifficultyLevel.objects.count() == 1
+    assert DifficultyLevel.objects.first().name == "hard"
```

Comment:
I would change this assertion to `assert DifficultyLevel.objects.first().name == difficulty_level.name`, because when someone changes the name in fixture then this test is going to fail.

Reply:
Since we are not doing custom save any more this test is no more required.